src/pages/HomePage.java

```diff
@@ -0,0 +1,32 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.PageFactory;
+
+public class HomePage extends BasePage{
+
+    BasePage page = new BasePage(driver);
+    private static String BASE_URL = "https://testkwidos.tk";
+
+    //locators
+    @FindBy(css = "ul.user.nav li")
+    WebElement signInButton;
+
+    public HomePage(WebDriver driver){
+        super(driver);
+        PageFactory.initElements(driver, this);
```

Comment:
There is no need to use this.

Reply:
@vladimirKorzhukov
If I remove it the code will not compile. or I won't be able to use @FindBy.